src/test/java/core/basesyntax/service/fileoperation/impl/CreateReportImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.fileoperation.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.FruitShopStorage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.fileoperation.CreateReport;
+import org.junit.After;
+import org.junit.Test;
+
+public class CreateReportImplTest {
+    private static final String TITLE = "fruit,quantity"
+            + System.lineSeparator();
+    private static final StringBuilder builder = new StringBuilder();
+    private static final StorageDao dao = new StorageDaoImpl();
+    private static final CreateReport report = new CreateReportImpl(builder, dao);
```

Comment:
```suggestion
    private static final StorageDao dao;
    private static final CreateReport report;
```
initialize in method with annotation @BeforeClass
and if you need StringBuilder just create it where you need it, don't pass it with constructor